src/X.PagedList/OrderedPagedList.cs

```diff
@@ -62,6 +62,6 @@ private void InitSubset(IEnumerable<T> superset, Func<T, TKey> keySelectorMethod
         var skip = (pageNumber - 1) * pageSize;
         var items = superset.OrderBy(keySelectorMethod).Skip(skip).Take(pageSize).ToList();
 
-        Subset.AddRange(items);
+        FillSubset(items);
```

Comment:
I would use just `Subset = items` here. No need for AddRange, we get a full proper list in items and do not use it elsewhere, so no need for a defensive copy.